cppguide.html

```diff
@@ -2201,23 +2143,33 @@ <h3 id="Rvalue_references">Rvalue References</h3>
 </ul>
 
 <p class="decision"></p>
-<p>You may use rvalue references to define move constructors and move
-assignment operators (as described in
-<a href="#Copyable_Movable_Types">Copyable and Movable Types</a>). See the
-<a href="primer#copying_moving">C++ Primer</a> for more information about
-move semantics and <code>std::move</code>.</p>
-
-<p>You may use rvalue references to define pairs of overloads, one taking
-<code>Foo&amp;&amp;</code> and the other taking <code>const Foo&amp;</code>.
-Usually the preferred solution is just to pass by value, but an overloaded pair
-of functions sometimes yields better performance and is sometimes necessary in
-generic code that needs to support a wide variety of types. As always: if
-you're writing more complicated code for the sake of performance, make sure you
-have evidence that it actually helps.</p>
-
-<p>You may use forwarding references in conjunction with <code>
-<a href="http://en.cppreference.com/w/cpp/utility/forward">std::forward</a></code>,
-to support perfect forwarding.</p>
+<p>Do not use rvalue references (or apply the <code>&amp;&amp;</code>
+qualifier to methods), except as follows:</p>
+<ul>
+  <li>You may use them to define move constructors and move assignment
+  operators (as described in
+  <a href="#Copyable_Movable_Types">Copyable and Movable Types</a>). See the
+  <a href="primer#copying_moving">C++ Primer</a> for more information about
```

Comment:
The primer is internal only, we should trim this sentence / link.

Reply:
Good catch—fixed!